### ISSUE TITLE
Internal terminal(Powershell) fails to render Nerd Fonts Properly

### ISSUE THREAD

**Issue by blackironj** [lensapp/lens#8150]
**Describe the bug**
I am using PowerShell in the internal terminal of Lens. In this case I think Nerd Fonts are not loading properly, causing the characters to appear broken.
I tried to use all kind of fonts, `( Anoymous Pro, IBM Plex Mono, Jetbrains Mono, Rd Hat Mono, RobotoMono, Source Code Pro, Space Mono, ubuntu Mono)`  but the same issue persists.

**Expected behavior**
In the external, I am using the `Roboto Mono Nerd Font`
![expected](https://github.com/user-attachments/assets/4937fd92-1b3c-48f4-a707-4bd081d43a6c)

**Screenshots**
![terminal-font](https://github.com/user-attachments/assets/6ecc310e-af82-45c2-964d-e469bdc6a286)

**Environment (please complete the following information):**
- Lens Version: 
   - Lens: 2024.11.261604-latest
   - Electron: 31.7.5
   - Chrome 126.0.6478.234
   - Node: 20.18.0
- OS: Windows11
- Powershell: 7.4.6

**Additional context**

**Comment by Tixon123** [comment]
Hello blackironj,

 Thank you for reaching out to Lens support!

 Thank you for reporting a bug. 

 We are working on your issue. Stand by for further updates.

Best Regards,
Tikhon Kudinov
Lens Support Engineer
Mirantis, Inc

**Comment by panuhorsmalahti** [comment]
https://github.com/ryanoasis/vim-devicons/issues/269 seems like similar issue as here, can you try some of the workarounds mentioned there, including trying "Hack Nerd Font".

**Comment by Ergamon** [comment]
I found this issue here. Basically I came here to ask for a feature request.

I am using PowerShell 7 with a customized prompt also relying on the special icons of Nerd Fonts.

Basically "the problem" is, that you can use any local shell (in my case pwsh) through the preferences, but you cannot use any local font. The dropdown in the preferences only allow you to specify a font included in the Lens installation.

So to resolve the problem of the issue creator (and of course mine ;)), I see 2 options:
Allow Lens to use installed fonts or package Lens with the use the Nerd Font patched versions:
https://www.nerdfonts.com/font-downloads

Basically I think you make a big part of your community happy by exchanging the fonts with the patched versions and for those not needing the extra icons I dont see any kind of drawback. But perhaps I am missing something?

**Comment by panuhorsmalahti** [comment]
From those options allowing Lens to use any installed font seems like the better choice.

**Comment by Tronyxsystem** [comment]
Hello,
Any news about this bug ?
